libs/api/domains/cms/src/lib/services.ts

```diff
@@ -233,6 +234,19 @@ export const getArticle = async (
   return result.items.map(mapArticle)[0] ?? null
 }
 
+export const getUrl = async (
+  slug: string,
+  lang: string,
+): Promise<Url | null> => {
+  const result = await getLocalizedEntries<types.IUrlFields>(lang, {
+    ['content_type']: 'url',
+    'fields.urlsList[all]': slug,
+    include: 10,
```

Comment:
This will fetch all related article to a depth of 10, which would make the response HUGE.